resources/views/pagination-url.blade.php

```diff
@@ -33,7 +33,7 @@ class="py-2 px-3 w-full bg-transparent dark:text-theme-secondary-200"
             class="button-pagination-page-indicator button-pagination-page-indicator--search"
             :class="{ 'opacity-0': search }"
             x-on:click="toggleSearch"
-        ><span>Page {{ $paginator->currentPage() }} of {{ $paginator->lastPage() }}</span></button>
+        ><span>@lang('ui::generic.pagination.current_to', ['currentPage' => $paginator->currentPage(), 'lastPage' => $paginator->lastPage()])</span></button>
```

Comment:
Not sure if the elements need to be next to each other or not so i left them, but the line was too long so split up the array

```suggestion
        ><span>@lang('ui::generic.pagination.current_to', [
        	'currentPage' => $paginator->currentPage(),
        	'lastPage' => $paginator->lastPage(),
        ])</span></button>
```